app/grandchallenge/forge/models.py

```diff
@@ -49,10 +53,16 @@ def is_file(self):
             and not self.relative_path.endswith(".json")
         )
 
+    @property
+    def is_dicom_image_set(self):
+        return (
+            self.super_kind == ForgeSuperKindEnum.IMAGE
+            and self.kind == ForgeKindEnum.DICOM_IMAGE_SET
```

Comment:
We could take these properties from GC rather than calculating them here (is_dicom_image_set corresponds to is_image - these correspond to is_dicom_image_kind, is_panimg_kind and is_image_kind in the rest of the GC codebase, having the separate is_image_kind might save some of those long if statements). Even if not, it is more likely that we extend the DICOM image types rather than the PANIMG types.

Reply:
Good idea. I'll trace them a bit since they might have drifted over time.